opencog/cython/opencog/nameserver.pyx

```diff
@@ -77,12 +77,12 @@ cdef create_python_value_from_c_value(cValuePtr& value, AtomSpace atomspace):
     cdef cValue *c_ptr = value.get()
     if clazz is not None:
         if c_ptr.is_atom():
-            return clazz(ptr_holder = ptr_holder, atomspace = atomspace)
+            return clazz(ptr_holder = ptr_holder)
```

Comment:
Branches are equal, so `if: else:` can be replaced by:
```py
return clazz(ptr_holder = ptr_holder)
```

Reply:
done